dart/pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   meta: ^1.3.0
   stack_trace: ^1.10.0
   uuid: '>=3.0.0 <5.0.0'
+  web: ^0.5.1
```

Comment:
Do we actually need to add this dependency? Tests passed locally regardless.
```suggestion
```

Reply:
Yeah, you have to leave it!

Reply:
The question is whether it actually works. We can ignore the warning for some time before cutting the compatibility.

Reply:
I don't quite understand why we can leave it out since we have to use the package

```dart
import 'package:web/web.dart' as web show window, Window, Navigator;

// and in the enricher event processor we use web.window
```